src/wallet/chialisp.py

```diff
@@ -69,8 +69,21 @@ def sha256(*argv):
     return apply("sha256", argv)
 
 
+# run -i clvm_runtime '(mod X (include sha256tree.clvm) (sha256tree0 X))'
+
+SHA256TREE_PROG = """
+
+((c (q ((c 2 (c 2 (c 3 (q ()))))))
+    (c (q ((c (i (l 5)
+                 (q (sha256 (q 2)
+                            ((c 2 (c 2 (c 9 (q ())))))
+                            ((c 2 (c 2 (c 13 (q ())))))))
+                 (q (sha256 (q 1) 5))) 1))) %s)))
+"""
+
+
```

Comment:
I think this may be a merge error as Richard removed this recently in dev IIRC

Reply:
I did one more rebase and I hope this is fixed as well

Reply:
Anyway, I saw the same version as we have in the dev branch.